pyproject.toml

```diff
@@ -44,28 +46,43 @@ virtualenv = [
     { version = "^20.4.3,!=20.4.5,!=20.4.6" },
     { version = "<20.16.4", markers = "sys_platform == 'win32'" },
 ]
+ale-py = { version = "~=0.8.1", optional = true }
+autorom = { version = "~=0.4.2", extras = ["accept-rom-license"], optional = true }
+box2d_py = { version = "2.3.5", optional = true }
+cython = { version = ">=0.27.2", optional = true }
 
 # TODO: add versions
-atari_py = { version = "*", optional = true }
-envpool = { version = "^0.8.2", optional = true }
-mujoco_py = { version = "*", optional = true }
+envpool = { version = "^0.8.2", optional = true,  markers = "sys_platform != 'darwin'"}
+gymnasium-robotics = { version = "*", optional = true }
+imageio = { version = ">=2.14.1", optional = true }
+mujoco = { version = ">=2.1.5", optional = true }
+mujoco-py = { version = ">=2.1,<2.2", optional = true }
 opencv_python = { version = "*", optional = true }
 pybullet = { version = "*", optional = true }
+pygame = { version = ">=2.1.3", optional = true }
+shimmy = { version = ">=0.1.0,<1.0", optional = true }
+swig = { version = "4.*", optional = true }
+vizdoom = { version = "*", optional = true }
 
 [tool.poetry.extras]
-atari = ["atari_py", "opencv-python"]
-mujoco = ["mujoco_py"]
+atari = ["ale-py", "autorom", "opencv-python", "shimmy"]
```

Comment:
Please explain in a block comment why we have to do it like that (with a link to the relevant poetry issue).

This part requires attention and monitoring gymnasium for changes